meta.thrift

```diff
@@ -0,0 +1,4 @@
+service Meta {
```

Comment:
This file should really live in tchannel and be imported from there rather then copied into two places.

Let's move it there in the future.